content/en/docs/how-tos/contributing-openshift-release.md

```diff
@@ -77,10 +77,23 @@ whenever a pull request would affect one or more CI jobs. Jobs affected by such
 target component repository after the changes would be merged. This provides pull request authors early feedback about
 how config changes impact CI setup.
 
-All pull requests trigger a `ci/prow/pj-rehearse` job that inspects the changes in the PR and detects affected jobs. It
-then submits these jobs for execution, and they will report to the pull request results via the GitHub contexts named
-with the `ci/rehearse/$org/$repo/$branch/$test` pattern. Both the "driver" job (`ci/prow/pj-rehearse`) and the individual
-rehearsals do not block merges. This allows merging changes to CI configuration that affect jobs that fail for reasons
+#### External Prow Plugin
+
+All pull requests trigger a `pj-rehearse` external prow plugin. It checks the modified files for any rehearsable changes,
+and comments a list of them. It also supplies directions for interacting with itself. The user can either comment:
+* `/pj-rehearse` to run up to 10 rehearsal jobs from the list
+* `/pj-rehearse more` to run up to 20 rehearsal jobs from the list
+* `/pj-rehearse max` to run up to 35 rehearsal jobs from the list
+* `/pj-rehearse skip` to opt-out of rehearsals for the PR
+* `/pj-rehearse ack` to acknowledge the rehearsals (pass or fail)
+
+Once the `rehearsals-ack` label is present on the PR it will be able to be merged once all additional merge criteria are met.
```

Comment:
```suggestion
Once the `rehearsals-ack` label is present on the PR it will be able to be merged provided that all additional merge criteria are met.
```